modules/ipi-install-configure-multiple-cluster-nodes.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+:_content-type: CONCEPT
+[id="ipi-install-configure-multiple-cluster-nodes_{context}"]
+= Configure multiple cluster nodes
+
+You can simultaneously configure {product-title} cluster nodes with identical settings. Configuring multiple cluster nodes avoids adding redundant information for each node to the `install-config.yaml` file. This file contains specific parameters to apply identical configuration to multiple nodes in the cluster.
+
+Worker nodes are configured separately from the controller node. However, configurations for both node types use specific host parameters in the `install-config.yaml` file, as shown in the following example:
+
+[source,yaml]
+----
+hosts:
+- name: ostest-master-0
+ [...]
+ networkConfig: &BOND
+   interfaces:
+   - name: bond0
+     type: bond
+     state: up
+     ipv4:
+       dhcp: true
+       enabled: true
+     link-aggregation:
+       mode: active-backup
+       port:
+       - enp2s0
+       - enp3s0
+- name: ostest-master-1
+ [...]
+ networkConfig: *BOND
+- name: ostest-master-2
+ [...]
+ networkConfig: *BOND
+----
+
+NOTE: Configuration of multiple cluster nodes is only available for Installer-provisioned Installation on initial deployments.
```

Comment:
IPI is "installer-provisioned infrastructure", but we also try not to sue that as a distinguishing term in docs. I'm not sure if there is another way to say it in this case or not. If not, I suggest:
```suggestion
[NOTE]
====
Configuration of multiple cluster nodes is only available for initial deployments on installer-provisioned infrastructure.
====
```